desktop/release-notes.md

```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
-## 4.21.1
+## 4.22.0
 
-{% include release-date.html date="2023-07-03" %}
+{% include release-date.html date="2023-07-27" %}
 
-{% include desktop-install.md all=true version="4.21.1" build_path="/" %}
+{% include desktop-install.md all=true version="4.22.0" build_path="/" %}
 
 ### Upgrades
-- [Compose v2.19.1](https://github.com/docker/compose/releases/tag/v2.19.1)
+
+- [Buildx v0.11.2](https://github.com/docker/buildx/releases/tag/v0.11.2)
+- [Compose v2.20.2](https://github.com/docker/compose/releases/tag/v2.20.2)
+- [Docker Engine v24.0.5](https://docs.docker.com/engine/release-notes/24.0/#2405)
+
+### New
+- [Resource Usage](use-desktop/container.md) has moved from experimental to GA.
+- You can now split large Compose projects into multiple sub-projects with [`include`](../compose/multiple-compose-files/include.md).
 
 ### Bug fixes and enhancements
 
+#### For all platforms
+- [Settings Management](hardened-desktop/settings-management/index.md) now lets you turn off Docker Extensions for your organisation.
+- Fixed a bug where turning on Kubernetes from the UI failed when the system was paused.
+- Fixed a bug where turning on Wasm from the UI failed when the system was paused.
+- Bind mounts are now shown when you [inspect a container](use-desktop/container.md).
+- You can now download Wasm runtimes when the containerd image store is enabled.
+
+#### For Mac
+
+- Fixed a bug that prevented Docker Desktop from starting. [docker/for-mac#6890](https://github.com/docker/for-mac/issues/6890)
+- Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```

Comment:
@aevesdocker @doringeman sorry for late feedback. i wouldn't lead with the fact that this is experimental, it's being rolled out to all users by default.

can we please adjust to:

"Resource Saver is now available on Mac and Windows, optimising Docker Desktop's usage of your system resources when no containers are running."

Reply:
I think we still need to highlight that you'll only see this feature if you have enabled access to experimental features. The current phrasing matches how we've introduced other experimental features in release notes, but how about 
"Resource Saver is now available on Mac and Windows. It optimises Docker Desktop's usage of your system resources when no containers are running. To access this feature, make sure you have enabled access to experimental features."

Reply:
I agree!

And use "It **massively** reduces"? 😁  https://github.com/docker/docs/pull/17821#discussion_r1282978170

Reply:
My only concern here is that we're labelling this as 'experimental' but it's going to be default enabled. But I guess that is the case for all experimental features (e.g. they're _all_ default-enabled), so my concern is perhaps invalid.

Happy to go with the wording you propose @aevesdocker , and also take your steer on whether 'massively' is the right language for the Release Notes. Though I appreciate the marketing @doringeman 😂